packages/application-components/src/components/modal-pages/form-modal-page/form-modal-page.mdx

```diff
@@ -52,14 +52,13 @@ import { FormModalPage } from '@commercetools-frontend/application-components';
 | `dataAttributesSecondaryButton` | `object`                   |    -     | -                         | Use this prop to pass `data-` attributes to the secondary button                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
 | `dataAttributesPrimaryButton`   | `object`                   |    -     | -                         | Use this prop to pass `data-` attributes to the primary button                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
 | `getParentSelector`             | `function`                 |    -     | -                         | The function should return an HTML element that will be used as the parent container to hold the modal DOM tree. If no function is provided, it's expected that an HTML element with the `id="parent-container"` is present in the DOM. In `NODE_ENV=test` environment, the default HTML element is `body`.                                                                                                                                                                                                         |
-| `customControls`                | `node`                     |    -     | -                         | Pass a React.node to be used in place of the pre-determined controls. This can be useful if you need actions other than Cancel & Confirm, or other types of buttons, while keeping the same modal header layout                                                                                                                                                                                                                                                                                                     |
```

Comment:
I guess this is considered a breaking change? 🤔

Reply:
Yep. And it's not the only one, since TabularModalPage also won't have defaults anymore. But we can discuss if we want to avoid these.